components/dialog/dialog-mixin.js

```diff
@@ -298,6 +305,28 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 		this._useNative = false;
 	}
 
+	_isCloseAborted() {
+
+		if (this._action === forceCloseAction) {
+			return false;
+		}
+		else if (this._action === abortAction || this._interceptDialogClosing) {
```

Comment:
Nit:
```suggestion
		} else if (this._action === abortAction || this._interceptDialogClosing) {
```

Reply:
Alternately, you could also invert this and add it to the first check which returns early:

```javascript
if (this._action === forceCloseAction || this._action !== abortAction) {
    return false;
}

const abortEvent = new CustomEvent('d2l-dialog-abort-close', {
	cancelable: true,
	detail: {
		forceCloseDialog: this._close.bind(this, forceCloseAction),
		action: this._action
	}
});
this.dispatchEvent(abortEvent);

return abortEvent.defaultPrevented;
```